tests/resources/test_files_resource.py

```diff
@@ -116,6 +117,8 @@ def test_files_api_flow(client, es_clear, headers, input_data, location):
     assert res.json['key'] == 'test.pdf'
     assert res.json['status'] == 'completed'
     assert res.json['metadata'] == {'title': 'Test file'}
+    file_size = str(res.json['size'])
+    assert bool(re.match(r'^(\d+)$', file_size)), "File size not integer"
```

Comment:
Minor: Better to explicitly assert that it is an integer type:

```suggestion
    assert isinstance(res.json['size'], int)
```